### PR TITLE
New package: bird[6]?-1.5.0

### PR DESCRIPTION
**Pull request by hanspolo**

### CODE REVIEW

File: srcpkgs/bird/files/bird/run [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec bird ${OPTS:=-c /etc/bird.conf -f}
```
Comment (Gottox) [inline srcpkgs/bird/files/bird/run]: Missing \n ;)

File: srcpkgs/bird6/bird6/files/bird6/run [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec bird ${OPTS:=-c /etc/bird.conf -f}
```
Comment (Gottox) [inline srcpkgs/bird6/bird6/files/bird6/run]: Missing \n ;)

File: srcpkgs/bird6/files/bird6/run [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec bird6 ${OPTS:=-c /etc/bird6.conf -f}
```
Comment (Gottox) [inline srcpkgs/bird6/files/bird6/run]: Missing \n ;)

File: srcpkgs/bird6/bird6/files/bird6/run (hunk 2 above)
Comment (Gottox) [inline srcpkgs/bird6/bird6/files/bird6/run]: ... and this dir is wrong (srcpkgs/bird6/bird6)

File: srcpkgs/bird6/bird6/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+homepage="http://bird.network.cz"
+distfiles="ftp://bird.network.cz/pub/$pkgname/$pkgname-$version.tar.gz"
+hostmakedepends="flex"
+makedepends="ncurses-devel readline-devel"
+conf_files="/etc/bird.conf"
+checksum=79d5c4254e25f5890626830ca42aae03735716bdf152ddd8368473d2b3c17b5d
+
+post_install() {
+	vsv bird
+}
```
Comment (Gottox) [inline srcpkgs/bird6/bird6/template]: Move this file to srcpkgs/bird6.

File: srcpkgs/bird/files/bird/run (hunk 1 above)
Comment (xtraeme) [inline srcpkgs/bird/files/bird/run]:
```
#!/bin/sh
[ -r conf ] && . ./conf
exec bird ${OPTS:=-c /etc/bird.conf} -f
```

File: srcpkgs/bird6/template [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+distfiles="ftp://bird.network.cz/pub/bird/bird-$version.tar.gz"
+wrksrc=bird-$version
+hostmakedepends="flex"
+makedepends="ncurses-devel readline-devel"
+configure_args="--enable-ipv6"
+conf_files="/etc/bird6.conf"
+checksum=79d5c4254e25f5890626830ca42aae03735716bdf152ddd8368473d2b3c17b5d
+
+post_install() {
+	vsv bird6
```
Comment (xtraeme) [inline srcpkgs/bird6/template]: If bird installs bird, and bird6 installs bird6 why do you need two packages? it's pointless.

Comment (xtraeme) [inline srcpkgs/bird6/template]: You can create a single pkg with both binaries/services, run the configure/build/install phase twice.

File: srcpkgs/bird/template [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+	# bird
+	make ${makejobs}
+
+	# bird6
+	cd bird6
+	make ${makejobs}
+}
+
+do_install() {
+	make DESTDIR=$DESTDIR install
```
Comment (xtraeme) [inline srcpkgs/bird/template]: what I meant was to provide both in a single pkg:

```
do_install() {
  make DESTDIR=${DESTDIR} install
  cd bird6
  make DESTDIR=${DESTDIR} install
}
```

**Comment by xtraeme** [comment]
Please make the following changes:

- add `/etc/bird{,6}.conf` to `conf_files`.
- add `--sbindir=/usr/bin` to `configure_args`.

**Comment by hanspolo** [comment]
Is the latest change correct?

**Comment by xtraeme** [comment]
Yes, this is what I meant, thanks :-)

**Pull request merged**